gulpfile.js

```diff
@@ -354,12 +356,10 @@ gulp.task('css', ['compile:css'], function () {
  * Watches for changes if --development flag is given
  */
 gulp.task('compile:css', function () {
-  var outputStyle = isDev ? 'expanded' : 'compressed';
-
   var tasks = rootCompileFolders.map(function(folder) {
     return gulp.src(folder + '/scss/**/*.scss')
       .pipe(sourcemaps.init())
-      .pipe(sass({ outputStyle: outputStyle })
+      .pipe(sass({ outputStyle: 'compressed' })
```

Comment:
This shouldn't have been squashed into a feature commit, but then again I should've raised it separately from the feature branch as well. The original commit message was:

Always use 'compressed' CSS compilation
    
    Which means you can commit artifacts created through 'npm run css --development'
    rather than 'npm run css' without recompiling them (since the output format is the same).
    
    We might want to separate this to 'npm run css --watch' and 'npm run css --debug'
    further down the road